cmd/payload-testing-prow-plugin/server.go

```diff
@@ -44,13 +46,32 @@ func helpProvider(_ []prowconfig.OrgRepo) (*pluginhelp.PluginHelp, error) {
 	return pluginHelp, nil
 }
 
+type trustedChecker interface {
+	trustedUser(author, org, repo string, num int) (bool, error)
+}
+
+type githubTrustedChecker struct {
+	githubClient github.Client
+	pluginAgent  *plugins.ConfigAgent
+}
+
+func (c *githubTrustedChecker) trustedUser(author, org, repo string, num int) (bool, error) {
```

Comment:
quick Q: Why do we need to use trigger config for that? Also, do we even have a configuration for the trigger plugin?

Reply:
The trigger's config is used in the function.
https://github.com/kubernetes/test-infra/blob/ee1e7c845bb25177944bb5de37545f007f529059/prow/plugins/trigger/trigger.go#L221-L223

IIUC, it allows more than collaborators.

We do have config for trigger:
https://github.com/kubernetes/test-infra/blob/0d86dc3aed9603385d3e91ad76f367cfce5dd513/prow/plugins/config.go#L90

Reply:
About the config, I mean that we don't have any `triggers` configuration in our prow config.

Reply:
By adding a `trigers` configuration to the plugin config, is there any chance that it will affect anything else?

Reply:
I think Nikos has a great point here - it looks like we are only using two fields from `trigger` config, but our config does not actually utilize them at all, anywhere. So we should be fine with using the basic org/repo collaborator check you get when `onlyOrgMembers=false` and `trustedOrg=""`, and that way we do not need `trigger`s config at all.

Reply:
I see.
Updating...

Reply:
Done.